docs/architecture/cloud-native/index.md

```diff
@@ -59,6 +59,10 @@ Editors:
 
 > **Maira Wenzel**, Program Manager, .NET team, Microsoft
 
+## Version
```

Comment:
remove the text on Line 56: `Learn more about eShopOnContainers` Looks like a merge issue?

Reply:
Darn, thought I got rid of this. Gone now.